setup.py

```diff
@@ -9,4 +9,4 @@
       author='Yaniv Bronhaim',
       author_email='ybronhei@redhat.com',
       url="https://github.com/emesika/RaananaTiraProject",
-      scripts=["rose-client", "rose-server", "rose-admin"],)
+      scripts=["rose-client", "rose-server", "rose-admin", "requirements.txt"],)
```

Comment:
requirements.txt is not a script, and should not be installed in the system.

lets ignore the issue of packages duplication for now, and handle only pipenv integration in this PR.